CommunityFridgeMapApi/dependencies/python/notification.py

```diff
@@ -0,0 +1,71 @@
+from db import DB_Response
+import re
+import boto3
+from botocore.exceptions import ClientError
+
+#https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/ses/client/send_email.html
+
+# def sendNotification(message, userId, lastNotified, email, phoneNumber):
+
+
+def send_sms(phone_number:str=None, message:str=None, region:str='us-east-1'):
+    
+    sns = boto3.client(
+        "sns",
+        region_name=region
+    )
+    if(not is_valid_phone_number(phone_number)):
+        return
+
+    response = sns.publish(
+        PhoneNumber=phone_number,
+        Message = message
+    )
+
+    return
+
+
+def send_email(
+    sender_email:str=None, to_addresses:list=None, cc_addresses:list=None, bcc_addresses:list=None,
+    subject:str=None, message:str=None, html_data:str=None, reply_to_addresses:list=None):
+
+    client = boto3.client('ses')
+
+    Source = sender_email
+
+    Destination = {
+            'ToAddresses': to_addresses,
+            'CcAddresses': cc_addresses,
+            'BccAddresses': bcc_addresses
+            }
+
+    Message= {
+            'Subject:': {
+                'Data': subject,
+                'Charset': 'UTF-8',
+                },
+            'Body': {
+                'Text': {
+                    'Data': message,
+                    'Charset': 'UTF-8'
+                    },
+                'Html':{
+                    'Data': html_data,
+                    'Charset': 'UTF-8',
+
+                    }
+                }
+            }
+    ReplyToAddresses = reply_to_addresses
+    ReturnPath = ''  #This is the email address that is addressed when emails fail
+    SourceArn = '' #Something about authorization, See docs  
+    ReturnPathArn = '' #Same thing, something about authorization
+
+    response = client.send_email(
```

Comment:
can you check what the response of sns.send_email is and return if sns.send_email was successful or not

Reply:
sns.send_email returns a MessageId.  This update returns the error and the status of the message send